packages/graph/hash_graph/README.md

```diff
@@ -1,5 +1,33 @@
 # The HASH Graph API
 
+## Run the Graph
```

Comment:
the readme one level up seems tobe similar to this but only contain postgres steps. Which Readme should we make use of? both contents seems to be related to the Graph API

Reply:
Yes, the READMEs are confusing, it's not really foreseeable what is written where. We should probably simply merge these REEADMEs in a follow-up PR.

The README one directory up is only for setting up the database, this README describes how to run the Graph on the existing database.

Reply:
Yes, let's consider this in a follow-up. I suppose it's not really urgent, though